sos/report/plugins/proxmox.py

```diff
@@ -0,0 +1,114 @@
+# Copyright (C) 2024 ORNESS/Ditrit Drien Breton <drien.breton@orness.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+import json
+from pathlib import Path
+
+from sos.report.plugins import Plugin, DebianPlugin, PluginOpt
+
+
+class Proxmox(Plugin, DebianPlugin):
+    """
+    This plugin will capture information about the system's
+    Proxmox Virtualization Environment.
+
+    It will collect information about the cluster, nodes, pools and
+    storage from the Proxmox API.
+    """
+    short_desc = 'Proxmox cluster information'
+
+    plugin_name = 'proxmox'
+    packages = ('proxmox-ve',)
+
+    option_list = [
+        PluginOpt('output_formats',
+                  desc='List of output formats to use '
+                       'for the commands separated by ":".',
```

Comment:
We usually use comma as items separator (e.g. `sos report -o kernel,logs` to enable only those two plugins). I am not against a different separator here, just noting that this is unusual (and depends on users' preferences what is more natural for them).

Reply:
We use commas for top-level options, we are _unable_ to use them in plugin options due to how argparse parses them vs. our expectation of slicing multiple different plugin options at once. Colon delimiters in plugin options is the normal path.

E.G. `sos report -o kernel,logs,proxmox` is fine, as is `sos report -o proxmox,logs -k proxmox.output_formats=json,logs.all=on`, but `sos report -o proxmox,logs -k proxmox.output_format=json,text` would not get parsed properly.